### PR TITLE
Chat: add stories to storybook

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: apps/react-storybook/stories/chat/data.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { SchedulerTypes } from 'devextreme-react/scheduler';
+import { ChatTypes } from 'devextreme-react/chat';
+
+export const author_id_1 = "c94c0e76-fb49-4b9b-8f07-9f93ed93b4f3";
+export const author_id_2 = "d16d1a4c-5c67-4e20-b70e-2991c22747c3";
+
+export const author_info_1: ChatTypes.User = {
```
Comment (ksercs) [inline apps/react-storybook/stories/chat/data.ts]: Why do you use a snake case?

File: apps/react-storybook/stories/chat/Chat.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            setPopupWithScrollViewVisible(true);
+        }, [setPopupWithScrollViewVisible]);
+        
+        const hide = useCallback(() => {
+            setPopupWithScrollViewVisible(false);
+        }, [setPopupWithScrollViewVisible]);
+        
+        return (
+            <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+                <Button
```
Comment (ksercs) [inline apps/react-storybook/stories/chat/Chat.stories.tsx]: Why have u added this button, not just render a visible popup?

File: apps/react-storybook/stories/chat/Chat.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                    text="Show Chat In Popup"
+                    width={300}
+                    type="default"
+                    onClick={showPopupWithScrollView}
+                />
+                
+                <Popup
+                    width={400}
+                    height={500}
```
Comment (ksercs) [inline apps/react-storybook/stories/chat/Chat.stories.tsx]: Please decrease dimensions to make the scenario more real

File: apps/react-storybook/stories/chat/Chat.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+    disabled: false,
+    rtlEnabled: false,
+    visible: true,
+    hint: undefined,
+    activeStateEnabled: true,
+    hoverStateEnabled: true,
+    focusStateEnabled: true,
+};
+
+export const Common: Story = {
```
Comment (ksercs) [inline apps/react-storybook/stories/chat/Chat.stories.tsx]:
```suggestion
export const Overview: Story = {
```

Comment (EugeniyKiyashko) [reply apps/react-storybook/stories/chat/Chat.stories.tsx]: https://github.com/DevExpress/DevExtreme/pull/28125/commits/b5af372a380ba8144e767de9a8417707b11ab838

File: apps/react-storybook/stories/chat/Chat.stories.tsx (hunk 3 above)
Comment (EugeniyKiyashko) [reply apps/react-storybook/stories/chat/Chat.stories.tsx]: https://github.com/DevExpress/DevExtreme/pull/28125/commits/b5af372a380ba8144e767de9a8417707b11ab838

Comment (EugeniyKiyashko) [reply apps/react-storybook/stories/chat/Chat.stories.tsx]: image

File: apps/react-storybook/stories/chat/data.ts (hunk 1 above)
Comment (EugeniyKiyashko) [reply apps/react-storybook/stories/chat/data.ts]: https://github.com/DevExpress/DevExtreme/pull/28125/commits/b5af372a380ba8144e767de9a8417707b11ab838

File: apps/react-storybook/stories/chat/Chat.stories.tsx (hunk 2 above)
Comment (EugeniyKiyashko) [reply apps/react-storybook/stories/chat/Chat.stories.tsx]: https://github.com/DevExpress/DevExtreme/pull/28125/commits/b5af372a380ba8144e767de9a8417707b11ab838

**Review by ksercs — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Review by Shuvaevmedia — APPROVED** [review]

**Pull request merged**